components/supervisor/pkg/supervisor/services.go

```diff
@@ -1133,6 +1133,7 @@ func (s *taskService) ListenToOutput(req *api.ListenToOutputRequest, srv api.Tas
 		if err == io.EOF {
 			if isClosed.Load() {
 				// We are done
+				log.Println("[prebuildlog]: closing because task state is done and we are done reading")
```

Comment:
Do we want to merge these log statements (honest question, am fine to do so)?

Reply:
I thought about it and figured it does no harm. AFAIK we don't persist these anyway, so it might just be useful for viewing logs in realtime.